framework/architecture/hibernate/listeners.rst

```diff
@@ -0,0 +1,129 @@
+Listeners
+=========
+
+This chapter explains the different kinds of listeners that exist, how and when they are called
+and how they can be registered.
+
+EntityFacadeListener
+--------------------
+
+An :java:ref:`EntityFacadeListener<ch.tocco.nice2.persist.entity.events.EntityFacadeListener>` is called
+immediately after a certain entity operation (e.g. ``EntityManager#create()`` or ``Entity#setValue()``) and therefore
+this listener is independent of any hibernate action.
```

Comment:
> hibernate action

I assume hibernate action is Hibernate terminology, might be worth mentioning what exactly it is (e.g. in the glossary using ```:term:`Hibernate action` ```).